Penteract/Assets/Scripts/VideoScene1Start.cpp

```diff
@@ -8,6 +8,8 @@
 #include "Components/UI/ComponentVideo.h"
 #include "Components/UI/ComponentCanvas.h"
 
+const char* VideoScene1Start::globalVariableKeyPlayVideoScene1 = "CameFromMainMenu";
```

Comment:
I propose you to have all these variables (More variables will appear after this one) in a header file defined as a constexpr to be allocated at compile time. Also, we can group it in different namespaces to maintain and order and coherence inside the header file. What do you think? The header could be named as Globals.h or GlobalVariables.h

Reply:
Vote for GlobalVariables.h
I think its a very good idea.
This way also we will be able to include only 1 .h and use as many global vars as we linke, instead of having to find the specific script that had the variable, and include several files if we are doing something complex